modules/serverless-rn-1-32-0.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies
+//
+// * about/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-32-0_{context}"]
+= Red Hat {ServerlessProductName} 1.32
+
+{ServerlessProductName} 1.32 is now available. New features, updates, and known issues that pertain to {ServerlessProductName} on {ocp-product-title} are included in the following notes.
+
+[id="new-features-1-32-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.11.
+* {ServerlessProductName} now uses Knative Eventing 1.11.
+* {ServerlessProductName} now uses Kourier 1.11.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.11.
+* {ServerlessProductName} now uses Knative for Apache Kafka 1.11.
+* The `kn func` CLI plug-in now uses `func` 1.11.
+
+* {ServerlessProductName} multi-tenancy with {SMProductShortName} is now available as a Technology Preview (TP) feature.
```

Comment:
In this list, the following changes happened in previous releases (and were already announced there). Why do we include them here as well?

`OpenShift Serverless multi-tenancy with Service Mesh is now available as a Technology Preview (TP) feature.
`
`OpenShift Serverless can now be installed and used on single-node OpenShift.
`
`Pipelines-as-code for OpenShift Serverless Functions is now available as a Technology Preview (TP).
`
`The NamespacedKafka annotation, which was a Technology Preview (TP) feature, is now deprecated in favor of the standard Kafka broker with no data plane isolation.`
(Also duplicated in the Known issues. Even if this needed to be included, it only needs to be included in New features, which is the correct section for this type of announcement. Known issues should only contain non-intentional or unavoidable flaws. This issue was deprecated on purpose.)

`You can now configure a persistent volume claim (PVC) for an existing PersistentVolume object to use with OpenShift Serverless Functions and it is now available as a Developer Preview.
`
`The DomainMapping v1alpha1 custom resource definition (CRD) is now deprecated. Use v1beta1 CRD instead.
`
`Integrating Red Hat OpenShift Service Mesh with Serverless when Kourier is enabled is now deprecated. Use net-istio instead of net-kourier for Service Mesh integration.`

Reply:
Okay, noted! I have removed them now